tests/programs.rs

```diff
@@ -1,5 +1,7 @@
 extern crate bincode;
 extern crate elf;
+#[macro_use]
```

Comment:
Just an fyi, this should no longer be needed in the latest Rust. You can now import macros as you would any other function.